dependencies/graknlabs/dependencies.bzl

```diff
@@ -63,7 +63,7 @@ def graknlabs_verification():
     git_repository(
         name = "graknlabs_verification",
         remote = "https://github.com/alexjpwalker/verification",
```

Comment:
are we still meant to depend on your fork?

Reply:
Yes - this is for ensuring that the tests, newly added in my fork, pass in CircleCI. After graknlabs/verification#78 is **merged**, `dependencies.bzl` should be updated to point at graknlabs/verification.